### ISSUE TITLE
Can't reproduce the 16-segments results

### ISSUE THREAD

**Issue by yzfly** [Phoenix1327/tea-action-recognition#4]
Hi, thank your for sharing your code!

I have prepared the datasets as TSM suggested, and  trained  TEA on somethingsomethingv1. But I can only get 
Best Prec@1: 47.726  , which is much worse than your results. My training command is:

```
python main.py something RGB \
     --arch tea50 --num_segments 16 --gpus 0 1 2 3 \
     --gd 20 --lr 0.01 --lr_steps 30 40 45 --epochs 50 \
     --batch-size 16 -j 16 --dropout 0.5 --consensus_type=avg --eval-freq=1 \
     --experiment_name=TEA --shift \
     --shift_div=8 --shift_place=blockres --npb
```

What's wrong here? Can you share your 16-segments train settings?

**Comment by Phoenix1327** [comment]
We have mentioned the hyper parameter settings in Appendix B.2 (https://arxiv.org/pdf/2004.01398.pdf). Please carefully refer to this section for training details of our TEA model.
`For Something-Something, the batch size, initial learning rate, weight decay, and dropout rate are set  to 64, 0.02, 5e-4 and 0.5 respectively.`
Our TEA model is trained on 8 P40 with 24G memory, and we follow the same setting for both 8-segments experiments and 16-segments experiments on STHV1 & V2.
In Appendix B.2, we have also mentioned the learning rate should match the batch size. Thus, when you reduce the batch size from 64 to 16, I suggest to change the corresponding learning rate to 0.02/4=0.005.

**Comment by yzfly** [comment]
Thank you, I will try it.

**Comment by shoutOutYangJie** [comment]
> Thank you, I will try it.

hi, did you reproduce the precision as paper?

**Comment by yzfly** [comment]
> > Thank you, I will try it.
> 
> hi, did you reproduce the precision as paper?

Yes, with the author's guide, the results are reproduced

**Comment by lishukai428** [comment]
> Hi, thank your for sharing your code!
> 
> I have prepared the datasets as TSM suggested, and trained TEA on somethingsomethingv1. But I can only get Best Prec@1: 47.726 , which is much worse than your results. My training command is:
> 
> ```
> python main.py something RGB \
>      --arch tea50 --num_segments 16 --gpus 0 1 2 3 \
>      --gd 20 --lr 0.01 --lr_steps 30 40 45 --epochs 50 \
>      --batch-size 16 -j 16 --dropout 0.5 --consensus_type=avg --eval-freq=1 \
>      --experiment_name=TEA --shift \
>      --shift_div=8 --shift_place=blockres --npb
> ```
> 
> What's wrong here? Can you share your 16-segments train settings?
Hi, can you share the something-something v1 dataset? I can not find it. 15753110895@163.com.  Thanks!

**Comment by sudonto** [comment]
@lishukai428 your lr and bs is not match according to the paper

**Comment by hanyamin** [comment]
@yzfly，hello, could you share  the TEA model pretrained on somethingsomethingv1?  2443884807@qq.com  Thanks!
